app/javascript/modules/listings/SharedHelpers.tsx

```diff
@@ -230,12 +235,18 @@ export const getImageCardProps = (
   }
 }
 
-export const getEventNote = (listingEvent: ListingEvent) => {
+export const getEventNote = (
+  listingEvent: ListingEvent,
+  translations: RailsTranslations,
+  fieldName: string
+) => {
```

Comment:
suggestion (non-blocking): Avoid modifying this function by passing translated text to this function, e.g.

```js
events={listing.Information_Sessions?.map((informationSession) => {
  return {
    dateString:
      informationSession.Date && localizedFormat(informationSession.Date, "LL"),
    timeString: getEventTimeString(informationSession),
    note: getEventNote(
      {
        ...informationSession,
        getTranslatedString(
          informationSession.Venue,
          listing.translations,
          `${informationSession.Id}.Information_Sessions.Venue__c`
        )
      },
    )
  }
})}
```